docs/api/standard_library_types.md

```diff
@@ -59,7 +59,9 @@ types:
       (if < `-2e10`or > `2e10`) since 1 January 1970
     * `str`; the following formats are accepted:
         * `YYYY-MM-DD[T]HH:MM[:SS[.ffffff]][Z or [±]HH[:]MM]`
+        * `YYYY-MM-DD` is accepted in lax mode, but not in strict mode
         * `int` or `float` as a string (assumed as Unix time)
+    * [`datetime.date`][] instances are accepted in lax mode, but not in strict mode
```

Comment:
Is this `[]` missing a link?

Reply:
Good question! We're using new sphinx links that work with this syntax to the stdlib docs :). So in short, no. You can see this active link here :) https://74596f4b.pydantic-docs2.pages.dev/api/standard_library_types/#datetimedatetime

Reply:
@Viicos,

Your docs work is coming in handy :).

Reply:
Yes actually ``[`datetime.date`][]`` is a shortcut for ``[`datetime.date`][datetime.date]``